static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -254,6 +263,14 @@ define([
             // anything we want to happen after displaying ads
             window.googletag.cmd.push(postDisplay);
 
+            // show sponsorship placeholder if adblock detected
+            var sponsorshipIdFound = isSponsorshipContainerTest();
```

Comment:
Do you just want to do this on the first element with a sponsorship ID, or would we just want to perform this operation against every match on the page? Just a thought.